google/cloud/aiplatform/featurestore/featurestore.py

```diff
@@ -301,24 +302,253 @@ def list_entity_types(
+        all entityTypes in this Featurestore will be deleted prior to featurestore deletion,
+        and all features in each entityType will be deleted prior to each entityType deletion.
+
+        WARNING: This deletion is permanent.
+
+        Args:
+            force (bool):
+                If set to true, any EntityTypes and
+                Features for this Featurestore will also
+                be deleted. (Otherwise, the request will
+                only work if the Featurestore has no
+                EntityTypes.)
+            sync (bool):
+                Whether to execute this deletion synchronously. If False, this method
+                will be executed in concurrent Future and any downstream object will
+                be immediately returned and synced when the Future has completed.
+        Raises:
+            FailedPrecondition: If entityTypes are created in this Featurestore and force = False.
+        """
+        _LOGGER.log_action_start_against_resource("Deleting", "", self)
+        lro = getattr(self.api_client, self._delete_method)(
+            name=self.resource_name, force=force
+        )
+        _LOGGER.log_action_started_against_resource_with_lro(
+            "Delete", "", self.__class__, lro
+        )
+        lro.result()
+        _LOGGER.log_action_completed_against_resource("deleted.", "", self)
+
+    @classmethod
+    @base.optional_sync()
+    def create(
+        cls,
+        featurestore_id: str,
+        online_store_fixed_node_count: Optional[int] = 1,
+        labels: Optional[Dict[str, str]] = None,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+        request_metadata: Optional[Sequence[Tuple[str, str]]] = (),
+        encryption_spec_key_name: Optional[str] = None,
+        sync: bool = True,
+    ) -> "Featurestore":
+        """Creates a Featurestore resource.
+
+        Example Usage:
+
+            my_entity_type = aiplatform.EntityType.create(
+                entity_type_id='my_entity_type_id',
+                featurestore_name='projects/123/locations/us-central1/featurestores/my_featurestore_id'
+            )
+            or
+            my_entity_type = aiplatform.EntityType.create(
+                entity_type_id='my_entity_type_id',
+                featurestore_name='my_featurestore_id',
+            )
+
+        Args:
+            featurestore_id (str):
+                Required. The ID to use for this Featurestore, which will
+                become the final component of the Featurestore's resource
+                name.
+
+                This value may be up to 60 characters, and valid characters
+                are ``[a-z0-9_]``. The first character cannot be a number.
+
+                The value must be unique within the project and location.
+            online_store_fixed_node_count (int):
+                Required. Config for online serving resources.
```

Comment:
Then I guess we should remove the Optional wrapper, right?

Reply:
Made it `Optional` to be consistent with defined in the the Args, where removed the 1 value, added more description to indicate 1 node count will be launched without specifying anything.

Reply:
Updated the `online_store_fixed_node_count` to `Required` with default to 1, also added more docstring description.
Service api at this moment requires `Node count must be at least 1.`

Reply:
See comment/question above.